src/vanilla/utils/proxyMap.ts

```diff
@@ -81,15 +81,13 @@ export function proxyMap<K, V>(entries?: Iterable<[K, V]> | undefined | null) {
       const map = getMapForThis(this)
       const index = map.get(key)
       if (index === undefined) {
-        // eslint-disable-next-line @typescript-eslint/no-unused-expressions
         this.epoch // touch property for tracking
```

Comment:
Should this be outside of the "if" block?
Does it cover the case when we delete the item afterwards?

Reply:
Let me check